onebusaway-frontend-webapp/src/main/webapp/WEB-INF/content/api/config.jspx

```diff
@@ -42,6 +42,7 @@ if(typeof OBA !== 'undefined' &amp;&amp; typeof OBA.Config !== 'undefined') {
 	OBA.Config.serviceAlertText='<s:property value="serviceAlertText" />';
 	OBA.Config.feedbackFormText='<s:property value="feedbackFormText" />';
 	OBA.Config.feedbackFormURL='<s:property value="feedbackFormURL" />';
+	OBA.Config.legendVisibility='<s:property value="true" default="true" />';
```

Comment:
Close.  This needs to map to a property in the ConfigAction, which then pulls from config.json.  This appears to be hard coded instead.